examples/cars_on_parking/run_generation.sh

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 source ../../venv/bin/activate
-python ../../main.py --json_files data1.json data2.json --output_view parking --output_explore parking.lkml --columnn_name data_column --sql_table_name parking_table
\ No newline at end of file
+python ../../main.py --json_files data1.json data2.json --output_view parking.view --output_explore parking.lkml --columnn_name data_column --sql_table_name parking_table  --handle_null_values_in_sql true
```

Comment:
now we changed the input for --output_view. the filename should be added without any extension, as the only correct Looker extension will be added automatically.